### ISSUE TITLE
Clarify License

### ISSUE THREAD

**Issue by sjl** [fisxoj/hyperspec#2]
The `.asd` file says the license of this library is `LLGPLv3+` but the `LICENSE` file appears to be the vanilla GPLv3.  Which license is this library using?

**Comment by fisxoj** [comment]
You caught me!

Do you know of a v3+ version with the franz preamble?  It was my original intention to just use the latest LGPL with the preamble, but it seems to mention version 2.1 specifically and I sort of gave up  and you found the result.

I've prepared a branch to change things to the LLGPLv2.1.  If you know anything about all this, I'd value your thoughts.  If not, I can merge that later this week.

**Comment by sjl** [comment]
No, I haven't seen a GPLv3 with the preamble either (I admit I haven't really searched very hard).  Some of the folks in #lisp on Freenode might know more.
